jsapp/js/components/submissions/tableBulkOptions.tsx

```diff
@@ -250,6 +261,17 @@ class TableBulkOptions extends React.Component<TableBulkOptionsProps> {
               className='table-meta__additional-text'
             />
           )}
+
+        {Object.keys(this.props.selectedRows).length > 0 &&
+          (userCan(PERMISSIONS_CODENAMES.delete_submissions, this.props.asset) ||
+            userCanPartially(PERMISSIONS_CODENAMES.delete_submissions, this.props.asset)) &&
+          this.getSelectedSubmissionsWithAttachments().length > 0 && (
+            <BulkDeleteMediaFiles
+              selectedSubmissions={this.selectedSubmissionsWithAttachments}
+              selectedRowIds={Object.keys(this.props.selectedRows)}
+              assetUid={this.props.asset.uid}
+            />
+          )}
       </bem.TableMeta__bulkOptions>
     )
   }
```

Comment:
Could you please delete `requestDownloadFile` from `AttachmentActionsDropdown`? I've just noticed that leftover function :bow: